### PR TITLE
[IMP] website: add a system fonts option and use it in the default theme

### PR DESCRIPTION
**Pull request by qsm-odoo**
To allow increasing page loading performance, websites can now choose to
use system fonts instead of using a google font. It is also the default
in the default theme of Odoo from now on. The option will automatically
be available for all themes whatever way they defined their available
fonts but will only be used as default if explicitly asked by the theme
(as done in the default theme).

task-2993054

### CODE REVIEW

**Comment by qsm-odoo** [comment]
@stefanorigano @rdeodoo Following discussion at https://github.com/odoo/odoo/pull/100606#discussion_r978889942, I made this diff https://github.com/odoo/odoo/compare/2237489e12af1aefe7cc5b23ff35a6d8b2dd6be1..7bbe6ca25d71a7e494aa77961a1d9003ed0518f4 (actually, the definition of the variable in web is now justified with this PR alone as we need it for the frontend when website is not installed :+1:).

**Review by rdeodoo — APPROVED** [review]
Nitpicking one: first commit message contains: "you had to **to**"

Non blocking one (we can merge as such in the meantime, as SRI need this ASAP, maybe that's a non debate anyway, see below):
Can't we make the new default font smaller and thinner?
Everything now looks bigger and bolder. The bolder change makes everything less beautiful IMO.
See those 2-3 pages comparison: https://watch.screencastify.com/v/3aClJCJvnzLuePujSCJG
I guess that since it depends of the system font, that wouldn't be that easy since it would not be true in every scenario (some system fonts might be looking fine or might even looks too small) , so we should just live with that, right?

**Comment by qsm-odoo** [comment]
> Nitpicking one: first commit message contains: "you had to to"

Fixed

> Can't we make the new default font smaller and thinner?
> Everything now looks bigger and bolder. The bolder change makes everything less beautiful IMO.

@stefanorigano already plans to change the stack order / use different font from that list once this is merged. Let's leave that up to him after this merge. Personally, for the ubuntu font, I think it is way cleaner with it. Matter of taste, but personally it's like 6/10 for the previous fonts, 9/10 for the ubuntu ones. Mainly just because I prefer bold titles (anyways, I think SRI already said he would change the font stack to support more lighter bold if I remember what he said correctly). As for the font size, same IMO: I would really not prefer smaller than this.

In any case, let's not forget: it's just for the default theme, so you probably never reach this by default when using Odoo online (and even if you do, you can change it if you don't like it). Then if the user explicitly chooses it via the editor panel... that's his choice.

**Review by rdeodoo — APPROVED** [review]
@robodoo r+ rebase-ff

> In any case, let's not forget: it's just for the default theme, so you probably never reach this by default when using Odoo online (and even if you do, you can change it if you don't like it). Then if the user explicitly chooses it via the editor panel... that's his choice.

Fair point. I always get a bit too focused on the community and/or basic website install result (which still matters but which indeed makes such "issues" less critical).

**Comment by stefanorigano** [comment]
Perfect, thanks 👍

**Pull request closed without merging**
